drivers/disk/usdhc.c

```diff
@@ -1135,10 +1146,12 @@ static int usdhc_wait_cmd_done(struct usdhc_priv *priv,
 			int_status = base->INT_STATUS;
 		}
 
-		if ((int_status & USDHC_INT_TUNING_ERR_FLAG) ||
-			(int_status & USDHC_INT_CMD_ERR_FLAG)) {
+		if (int_status & USDHC_INT_TUNING_ERR_FLAG) {
 			error = -EIO;
 		}
+		if (int_status & USDHC_INT_CMD_ERR_FLAG) {
+			error = -ETIMEDOUT;
+		}
```

Comment:
is EIO and ETIMEDOUT handled differently by the caller? Curious why this change?

Reply:
They aren't to my knowledge. This change slipped in from debugging issues with the 1.8V support in the driver (since the logged return code can be useful). I've removed it.